statsmodels/multivariate/tests/test_multivariate_ols.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+import pandas as pd
+from statsmodels.multivariate.multivariate_ols import _MultivariateOLS
+from numpy.testing import assert_array_almost_equal, assert_raises_regex
```

Comment:
assert_raises_regex might need a version check
AFAIK, it's relatively new in numpy 
I don't know if we can use this in statsmodels 0.9